nessie/src/test/java/org/apache/iceberg/nessie/TestBranchVisibility.java

```diff
@@ -455,7 +456,9 @@ public void testWithRefAndHash() throws NessieConflictException, NessieNotFoundE
     String hashBeforeNamespaceCreation = api.getReference().refName(testBranch).get().getHash();
     Namespace namespaceA = Namespace.of("a");
     Namespace namespaceAB = Namespace.of("a", "b");
-    assertThat(nessieCatalog.listNamespaces(namespaceAB)).isEmpty();
+    NessieCatalog finalNessieCatalog = nessieCatalog;
```

Comment:
I don't think this is needed. You can just inline this and use `nessieCatalog.listNamespaces(namespaceAB)` directly